### DISCUSSION TITLE
Docker container update

### DISCUSSION THREAD

**Discussion by chris-fp** [nzbgetcom/nzbget#130] [Q&A]
As this is a fork from the original, Do we know when/if this fork will get "containerised"?

**Comment by the0ckid82** [comment]
https://hub.docker.com/r/mouton99/nzbget-ng

**Reply by ovizii** [reply]
I'm interested how to transition from the linuxserver.io package to this new fork. Any pointers are appreciated.

**Reply by Mouton99** [reply]
Simply replace the image URL/name you are using; use `mouton99/nzbget-ng:latest` instead of `lscr.io/linuxserver/nzbget:latest` (or similar).

**Reply by ovizii** [reply]
Thanks, I'll give it a try. I didn't think the mounted volumes and environmental variables would work.

**Reply by pipsen** [reply]
@Mouton99 Thank you for your great work! 
Any idea how to get rid of the annoying "update" message after each connect to Webinterface?

**Reply by Mouton99** [reply]
The version detection code from nzbget is broken in nzbget-ng I think, because of changes to `autoconfigure.ac` or something around that. It thinks the current version is `21.0.0`, which I guess is the default it uses when it can't detect the correct version with whatever mechanism it (tries to) use.  
I tried to fix it, but couldn't find how to.

**Reply by pipsen** [reply]
@Mouton99 
Check:
https://github.com/nzbget-ng/nzbget/discussions/27

@beertje44 managed it
https://hub.docker.com/repository/docker/jb044/nzbget-ng/general

Could you maybe take over and merge to your image?
![nzbget](https://github.com/nzbget-ng/nzbget/assets/15525250/12f1d4d3-8608-475e-89c5-5782ec593602)

**Reply by Mouton99** [reply]
meh. This repo here will be archived soon. See https://www.reddit.com/r/nzbget/comments/18ywnrf/nzbgetngnzbget_will_be_merging_into/
Better to switch to the Docker image provided by nzbgetcom [here](https://github.com/nzbgetcom/nzbget/blob/develop/docker/README.md).

**Reply by paul-chambers** [reply]
'meh'?

I never said I'd be archiving `nzbget-ng/nzbget`, and certainly won't until I've made sure everything of value and still relevant has made it into nzbgetcom/nzbget. I _hate_ seeing good work being dropped on the floor.

I'm in the process of moving all the issues and discussions in `nzbget-ng/nzbget` over to `nzbgetcom/nzbget` as we speak. I'll also be looking at those improvements in nzbget-ng that nzbgetcom hasn't already adopted, and making pull requests. No commitments on when though, due to the same problem I've been transparent about from the get-go - not enough time. I'm not a member of that repo either, so the final say over what is incorporated will be up to the maintainers of that repo.

For all the appreciation I have for hugbug and the large amount of time and effort he put into nzbget, I find it hard to understand why it was abandoned it so abruptly, and almost killing it by doing so. Baffling. 

Transitions **can** be handled gracefully, and that's my goal. `nzbget `will finally be back in the hands of people who can dedicate enough time to do it justice.
